x/fantoken/client/cli/cli_test.go

```diff
@@ -57,8 +58,8 @@ func (s *IntegrationTestSuite) TestToken() {
 
 	from := val.Address
 	symbol := "kitty"
-	denom := "ukitty"
 	name := "Kitty Token"
+	denom := strings.Replace(common.BytesToHash([]byte(from.String()+symbol+name)).Hex(), "0x", "ft", 1)
```

Comment:
Replace the denom function, instead to use `go-ethereum/common` use `github.com/tendermint/tendermint/crypto`.

*Example*
```
crypto.AddressHash([]byte(from.String()+symbol+name))
```

And then you can group all in a function (for example types.GetFantokenDenom(creator, denom, name) )